docs/src/modules/components/Demo.js

```diff
@@ -24,7 +24,8 @@ const styles = theme => ({
     },
   },
   demo: theme.mixins.gutters({
-    backgroundColor: theme.palette.background.contentFrame,
+    backgroundColor:
+      theme.palette.type === 'light' ? theme.palette.grey[200] : theme.palette.grey[900],
```

Comment:
I'm not convinced `theme.palette.grey` should exist in the default theme. Two components (Button & ExpansionPanelSummary) are using it for a background color when that should be in `theme.palette.background`, and one, Switch, should probably be calculating its color some other way (I didn't look further).

In the docs we can just import `grey` directly.

Reply:
grey existe in the default theme because we already import it. It's mainly here for the ease of use. Let's see if we can remove it from the core components to start with :) .

Reply:
> grey existe in the default theme because we already import it

I know, I"m saying we shouldn't.

> Let's see if we can remove it from the core components

Sure, but in that case, lets not add it here.

Reply:
I won't remove the grey usage. I remember why we inject it now: so people can provide their own tones.

Reply:
@oliviertassinari. If we fix the couple of components using this to use more appropriate theme keys, users can still change them there.

Don't worry if you don't want to fix it here. I can handle it in a separate PR. One more line to fix in the docs won't make a difference.

Reply:
> If we fix the couple of components using this to use more appropriate theme keys

Same I have been going the other way around. I want the palette to be as slim as possible. It's not a hub for all the configuration variable the components might need (we gonna need another solution for this). I think that we should aim for a palette as slim as possible. People shouldn't pay tooltip variable cost when only using a button. Also, it won't scale to an external component. Code splitting is important.

Reply:
> the palette isn't a hub for all the configuration variables the components might need 

Agreed.

> (we gonna need another solution for this)

You've already [documented](https://material-ui-next.com/customization/overrides/#global-theme-override) a couple. Do we need anything more?

Reply:
> You've already documented a couple. Do we need anything more?

@mbrookes I have been wondering about since 2016 when @nathanmarks redesigned the styling & theming solution. I don't have the answer 🤔 .